Source/WebKit/WPEPlatform/wpe/WPEView.cpp

```diff
@@ -1000,3 +1014,38 @@ void wpe_view_set_opaque_rectangles(WPEView* view, WPERectangle* rects, guint re
     if (viewClass->set_opaque_rectangles)
         viewClass->set_opaque_rectangles(view, rects, rectsCount);
 }
+
+/**
+ * wpe_view_set_gesture_controller:
+ * @view: a #WPEView
+ * @controller: a #WPEGestureController
+ *
+ * Set @controller as #WPEGestureController of @view
+ */
+void wpe_view_set_gesture_controller(WPEView* view, WPEGestureController* controller)
+{
+    g_return_if_fail(WPE_IS_VIEW(view));
```

Comment:
Add another g_return macro to check that controller is either null or `WPE_IS_GESTURE_CONTROLLER`